src/mn_setup.c

```diff
@@ -2082,6 +2084,10 @@ static setup_menu_t gen_settings2[] = {
     // [FG] play sounds in full length
     {"Disable Sound Cutoffs", S_ONOFF, M_X, M_SPC, {"full_sounds"}},
 
+    {"Resampler", S_CHOICE | S_NEXT_LINE, M_X, M_SPC, {"snd_resampler"}, m_null,
+     input_null, str_resampler, I_OAL_SetResampler},
```

Comment:
I'm violating the "encapsulation" of `i_oalsound.c/.h` but I don't want to deal with the function pointers in `i_sound.h` anymore.

Reply:
To have real "encapsulation" we need Choco-like config binding system. Maybe it's worth implementing to get rid of `extern` in `m_config`.